int64 x
int64 y
int64 start_x
int64 start_y
int64 finish_x
int64 finish_y
int64 kol_obst
int32[] obstacles
